package.json

```diff
@@ -0,0 +1,120 @@
+{
+  "name": "@logdna/env-config",
+  "version": "0.0.0",
+  "description": "Configuration package for reading environment variables",
+  "main": "index.js",
+  "scripts": {
+    "test": "tap",
+    "lint": "eslint ./",
+    "pretest": "npm run lint"
+  },
+  "bin": {
+    "config-doc": "./bin/cmd.js"
+  },
+  "keywords": [
+    "env",
+    "env var",
+    "environemnt",
+    "config",
+    "validate",
+    "validator",
+    "validation",
+    "parser"
+  ],
+  "bugs": {
+    "url": "https://github.com/logdna/env-config-node/issues"
+  },
+  "private": false,
+  "publishConfig": {
+    "access": "public"
+  },
+  "repository": {
+    "url": "https://github.com/logdna/env-config-node"
+  },
+  "homepage": "https://github.com/logdna/env-config-node",
+  "author": "LogDNA, Inc.",
+  "license": "MIT",
+  "eslintConfig": {
+    "root": true,
+    "ignorePatterns": [
+      "node_modules/",
+      "coverage/"
+    ],
+    "extends": [
+      "logdna"
+    ],
+    "parserOptions": {
+      "ecmaVersion": 2019
+    },
+    "rules": {
+      "no-shadow": 0,
+      "no-magic-numbers": 0,
+      "no-new": 0
+    }
+  },
+  "devDependencies": {
+    "@commitlint/cli": "^11.0.0",
+    "@commitlint/config-conventional": "^11.0.0",
+    "eslint": "^7.10.0",
+    "eslint-config-logdna": "^1.0.1",
+    "husky": "^4.3.0",
+    "tap": "^14.10.7"
+  },
+  "husky": {
```

Comment:
No reason, really.  Husky's good for using other hooks too. I think it also enables use of `commitlint` within `package.json`, otherwise I think you have to provide a separate `.conf` file for the rules.

I had never used these before, and it works this way.  We can do it a different way if `husky` is overkill, but `commitlint` will require some parameters for it to work that I haven't played with yet.

Reply:
After talking with @esatterwhite , I'll look into a commit template for this instead.  This is all learning curve--we need something to protect us from public PRs that have crappy commit messages.

Reply:
Actually, I think it should be a combination because a template doesn't enforce anything.  A template could certainly help outline what the expectation is, but `commitlint` I think is needed.

Reply:
worth noting that `husky` just sets up the git hook, since those can't be committed. whether we "install" that hook with `husky` or do it manually ourselves in a npm `postinstall` script, we still have to do it if we want git hooks (not a huge fan of them either, though). 

adding `commitlint` as part of CI linting seems reasonable, along with a template. my 2c.

Reply:
Yeah, it's kind of all the same, just different ways.  Even the [commitlint instructions](https://github.com/conventional-changelog/commitlint#getting-started) say to use `husky`.  It's a dev dependency, and it makes the hook obvious in the `package.json` as opposed to us burying it in `.git/hooks`.  I'd like to keep it as-is for now.  

I'll look into whipping up a template that highlights the rules and expected format though.